### ISSUE TITLE
Add '@' and '$' operators

### ISSUE THREAD

**Issue by dan1wang** [atom/language-pegjs#8]
See https://github.com/pegjs/pegjs/issues/235

**Comment by rsese** [comment]
Thanks for the report - I think the issue template might not have been available on this repository when you first opened the issue (it's been added since then), but can you summarize the issue and provide a minimal example that demonstrates the problem?

**Comment by StoneCypher** [comment]
Basically, the new maintainer of pegjs merged unnecessary operators to a development branch that was never published, two years ago, and a month ago announced that that branch is defunct and will never be released.

There's nothing to add.
